### DISCUSSION TITLE
Disambiguation code example, why prefer sea-orm or diesel over the other library

### DISCUSSION THREAD

**Discussion by mohs8421** [SeaQL/sea-orm#1499] [General]
## Motivation
We had some questions regarding the differences between sea-orm and diesel are. While there is an article on the manual about this question, it does not go into detail to allow new users to understand these differences and why one would choose for this or that library. Adding a directly visible code example would help here.

## Proposed Solutions
Add examples to the mentioned article, one for when to use diesel, and one for sea-orm

**Comment by billy1624** [comment]
Hey @mohs8421, thanks for the filing! As said on Discord, I'd open this up and see if anyone what to pick this up and contribute. If no, then I'll take hop in and work on it later. Thanks!!

**Comment by dotdot0** [comment]
@billy1624 Can I work on this one? It seems like it doesn't require much experience

**Comment by billy1624** [comment]
Hey @pratushrai0309, I think this is a good starting point!! Let me know if you need help :)

Some reference and resource for you.
- https://www.sea-ql.org/sea-orm-tutorial
- https://www.sea-ql.org/SeaORM
- https://github.com/SeaQL/sea-orm/tree/master/examples
- https://github.com/diesel-rs/diesel/tree/master/diesel_bench/benches/sea_orm_benches

**Comment by billy1624** [comment]
Btw... I think these two issues are for new contributors. And it's relative simpler compare to this issue.

- https://github.com/SeaQL/sea-orm/issues/826
- https://github.com/SeaQL/sea-orm/issues/888

**Comment by dotdot0** [comment]
@billy1624 Ok, I am gonna try them first
